### PR TITLE
lmdb: new package at 0.9.29

### PR DESCRIPTION
**Pull request by git-bruh**
https://termbin.com/bav2

### CODE REVIEW

File: community/lmdb/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+cd libraries/liblmdb
+
+# shellcheck disable=2086
+make CC=${CC:-cc}
```
Comment (dilyn-corner) [inline community/lmdb/build:6]: CC="${CC:-cc}"

Comment (dilyn-corner) [reply community/lmdb/build:6]: You could also replace the cd with a `make -C libraries/liblmdb` if you want.

Comment (git-bruh) [reply community/lmdb/build:6]: Ah i forgot that splitting cc is required only if you're invoking it directly.

make -C is not portable right ? Does bmake support it ?

Comment (dilyn-corner) [reply community/lmdb/build]: It does, but it's fine by me either way.

**Pull request merged**